webapp/components/interop-dashboard.js

```diff
@@ -672,5 +708,74 @@ class InteropDashboard extends PolymerElement {
     this.$.toggleStable.setAttribute('aria-pressed', true);
     this.$.toggleExperimental.setAttribute('aria-pressed', false);
   }
+
+  getSortIcon(index) {
+    if (this.toggleStates[index]) {
+      return '/static/expand_more.svg';
+    }
+    return '/static/expand_less.svg';
+  }
```

Comment:
This method technically takes in `sortToggled` but it is not listed explicitly as an argument. I think you should get the same functionality by passing the new `sortColumn` variable to this method.

Reply:
I reworked this one a bit but it might be a little convoluted still. Let me know if you think it can be further simplified. I wasn't able to get the correct behavior for double clicking a toggle without passing in `sortColumn` as well as `isSortedAsc`

Reply:
Yes, that sounds about right - you'll need them passed as arguments so that Polymer knows they have changed, and that it needs to re-evaluate what the value is.